Assets/ImageEffectGraph/Runtime/PostProcessing/GameModeOnlyParameters.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering.PostProcessing;
+
+namespace ImageEffectGraph.PostProcessing
+{
+    [Serializable]
+    public sealed class GameModeOnlyParameter : ParameterOverride<bool>

```

Comment:
Is there a particular reason this class is needed? It looks to me like a `BoolParameter` would do as well without adding another class.

Reply:
It is because I don't know they have this class :)
Fixed.